### DISCUSSION TITLE
How to get latex to work?

### DISCUSSION THREAD

**Discussion by zarte13** [mckaywrigley/chatbot-ui#1799] [Help]
How do I get the chatbot to show use latex to show equations? Right now it looks like raw latex Text, but doesn't show in nice looking equations.

**Comment by nefariousOne** [comment]
Were you able to get any help on this? 

I'm working on an older version of this, without supabase, and currently Latex/katex rendering is not working. I tried replacing katex with Mathjax but that didn't make any difference.  The correct node modules are installed and I'm referencing
`rehypePlugins={[rehypeMathjax]}`   within the ReactMarkdown component. It feels like there's a hook I'm missing somewhere? 

Any help out there?
